src/config.rs

```diff
@@ -176,13 +176,22 @@ pub struct Shell<'a> {
 }
 
 impl<'a> Shell<'a> {
-    pub fn new(program: &'a str) -> Shell<'a> {
+    pub fn new<S>(program: S) -> Shell<'a>
+            where S: Into<Cow<'a, str>> {
```

Comment:
`where` should be indented only 4 spaces, and the brace should be on the following line matching the indentation of `pub`.